app.js

```diff
@@ -1,7 +1,316 @@
+  popupCardContentDIV.className = 'popup-card-div';
+  popupCardContent.appendChild(popupCardContentDIV);
+
+  const popupCardTechnologiesList = document.createElement('ul');
+  popupCardTechnologiesList.className = 'butonns-languages';
+  popupCardContentDIV.appendChild(popupCardTechnologiesList);
+
+  myProjects[i].technologies.forEach((element) => {
+    const popupCardTechnologie = document.createElement('li');
+    popupCardTechnologie.className = 'btn-lg';
+    popupCardTechnologie.textContent = element;
+    popupCardTechnologiesList.appendChild(popupCardTechnologie);
+  });
+
+  const popupButtonDIV = document.createElement('div');
+  popupButtonDIV.className = 'popup-button-div';
+  popupCardContentDIV.appendChild(popupButtonDIV);
+  const popupButtonLiveLink = document.createElement('a');
+  popupButtonLiveLink.className = 'btn-project';
+  popupButtonLiveLink.textContent = 'See Live';
+  popupButtonLiveLink.href = '#';
+  popupButtonDIV.appendChild(popupButtonLiveLink);
+  const popupButtonLiveIcon = document.createElement('img');
+  popupButtonLiveIcon.src = 'images/icon-live.png';
+  popupButtonLiveLink.appendChild(popupButtonLiveIcon);
+  const popupButtonGithubLink = document.createElement('a');
+  popupButtonGithubLink.textContent = 'See Source';
+  popupButtonGithubLink.href = '#';
+  popupButtonGithubLink.className = 'btn-project';
+  popupButtonDIV.appendChild(popupButtonGithubLink);
+  const popupButtonGithubIcon = document.createElement('img');
+  popupButtonGithubIcon.src = 'images/icon-github-details.png';
+  popupButtonGithubLink.appendChild(popupButtonGithubIcon);
+}
+
+for (let i = 0; i < myProjects.length; i += 1) {
+    const cardList = document.createElement('li');
+    cardList.className = 'card';
+    cardUL.appendChild(cardList);
+  
+    const cardImage = document.createElement('img');
+    cardImage.className = `card-image${i}`;
+    cardImage.src = myProjects[i].image.link;
+    cardImage.altText = myProjects[i].image.altText;
+    cardList.appendChild(cardImage);
+  
+    const cardContent = document.createElement('div');
+    cardContent.className = 'card-content';
+    cardList.appendChild(cardContent);
+  
+    const cardTitle = document.createElement('h2');
+    cardTitle.className = 'card-title';
+    cardTitle.textContent = myProjects[i].title;
+    cardContent.appendChild(cardTitle);
+  
+    const cardDescription = document.createElement('p');
+    cardDescription.className = 'card-description';
+    cardDescription.textContent = `${myProjects[i].descriptionName}`;
+    cardContent.appendChild(cardDescription);
+  
+    const cardSpan = document.createElement('span');
+    cardSpan.textContent = `${myProjects[i].descriptionPosition} `;
+    cardSpan.className = 'dot';
+    cardDescription.appendChild(cardSpan);
+  
+    const cardSpanTwo = document.createElement('span');
+    cardSpanTwo.textContent = `${myProjects[i].descriptionYear}`;
+    cardDescription.appendChild(cardSpanTwo);
```

Comment:
[OPTIONAL]

- [ ]  This part of code and code block between lines 188 and 200 is almost the same code. (I am giving this part as an example, you have lots of repeated code in your js file.)
You should always follow the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and keep your code DRY(don't repeat yourself).
It’s always a nice idea to extract common logic into functions, so you can reuse them.
because having the same code in different places makes maintainability harder, once changes in the code will have to happen in many places, instead of one